schema/data/dev/004_cif_project.sql

```diff
@@ -2,10 +2,12 @@ begin;
 
 do $$
   declare current_revision cif.project_revision;
+  declare random_sector_name text;
```

Comment:
This dev data is used in the cypress tests, so I don't think we want sector to be randomly assigned. Any sector-related assertions could break, and we'll get happo diffs. Probably all the projects can have the same sector? They all have the same operator, summary, etc.